rts/Rendering/Models/3DModel.h

```diff
@@ -19,10 +20,12 @@
 #include "System/SpringMath.h"
 #include "System/creg/creg_cond.h"
 
-constexpr int MAX_MODEL_OBJECTS  = 3840;
-constexpr int AVG_MODEL_PIECES   = 16; // as it used to be
-constexpr int NUM_MODEL_TEXTURES = 2;
-constexpr int NUM_MODEL_UVCHANNS = 2;
+static constexpr int MAX_MODEL_OBJECTS  = 3840;
+static constexpr int AVG_MODEL_PIECES   = 16; // as it used to be
+static constexpr int NUM_MODEL_TEXTURES = 2;
+static constexpr int NUM_MODEL_UVCHANNS = 2;
+static constexpr int MAX_PIECES_PER_MODEL = std::numeric_limits<uint16_t>::max() - 1;
```

Comment:
Could document why -1 (not sure if I got it right here)
```suggestion
static constexpr int MAX_PIECES_PER_MODEL = std::numeric_limits<uint16_t>::max() - 1; // 0xFFFF reserved for "vertex not attached to any bone" magic value
```